backend/entities/office_hours/oh_ticket_entity.py

```diff
@@ -58,9 +62,75 @@ class OfficeHoursTicketEntity(EntityBase):
     )
 
     # One-to-one relationship of OfficeHoursTicket to UTA that has called it; optional field
-    caller_id: Mapped[int] = mapped_column(
+    caller_id: Mapped[int | None] = mapped_column(
         ForeignKey("academics__user_section.id"), nullable=True
     )
     caller: Mapped["SectionMemberEntity"] = relationship(
         back_populates="called_tickets"
     )
+
+    @classmethod
+    def from_model(cls, model: OfficeHoursTicket) -> Self:
+        """
+        Class method that converts an `OfficeHoursTicket` model into a `OfficeHoursTicketEntity`
+
+        Parameters:
+            - model (OfficeHoursTicket): Model to convert into an entity
+        Returns:
+            OfficeHoursTicketEntity: Entity created from model
+        """
+        return cls(
+            id=model.id,
+            caller_id=model.caller_id,
+            have_concerns=model.have_concerns,
+            caller_notes=model.caller_notes,
+            office_hours_event_id=model.office_hours_event_id,
+            description=model.description,
+            type=model.type,
+            state=model.state,
+            created_at=model.created_at,
+            closed_at=model.closed_at
+        )
+
+    def to_model(self) -> OfficeHoursTicket:
+        """
+        Converts a `OfficeHoursTicketEntity` object into a `OfficeHoursTicket` model object
+
+        Returns:
+            OfficeHoursTicket: `OfficeHoursTicket` object from the entity
+        """
+        return OfficeHoursTicket(
+            id=self.id,
+            caller_id=self.caller_id,
+            have_concerns=self.have_concerns,
+            caller_notes=self.caller_notes,
+            office_hours_event_id=self.office_hours_event_id,
```

Comment:
To follow up the previous, then it would be `self.event_id` instead!

Reply:
This has been updated, thank you!! (changed to oh_event_id=self.oh_event_id)